install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -129,18 +129,50 @@ It is recommended to deploy both groups to avoid potential impacts on
 performance in I/O and volume creation. Both of these are defined in the
 inventory hosts file.
 
-The definition of the clusters is done by including the relevant names in the
-`[OSEv3:children]` group, creating similarly named groups, and then populating
-the groups with the node information. The clusters can then be configured
-through a variety of variables in the `[OSEv3:vars]` group. `glusterfs`
-variables begin with `openshift_storage_glusterfs_` and `glusterfs_registry`
-variables begin with `openshift_storage_glusterfs_registry_`. A few other
-variables, such as `openshift_hosted_registry_storage_kind`, interact with the
-GlusterFS clusters.
-
-To prevent {gluster} pods from upgrading after an outage leading to a cluster
-with different {gluster} versions, it is recommended to specify the image name
-and version tags for all containerized components. The relevant variables are:
+To define the storage clusters, include the relevant names in the
```

Comment:
Please include a sample. I think you're saying that you put the cluster names in the `[OSEv3:children]` group and then create more `[OSEv3:<cluster_name>]` groups to define the variables for each cluster component, but I am not sure.

Reply:
I have asked QE reviewer if these lines are correct and to provide an example. If they are not correct, to please explain. Waiting to hear back from them.

Reply:
This is the sample inventory file which we use it for deployment.

https://docs.google.com/document/d/1-WyKSDz4xPNLEYYrptbW3G9jAZDAlRH61WusYr3-e9c/edit?usp=sharing

Reply:
Added example. Thank you.